helper.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import os
+import glob
+import sys
+import pickle
+from datetime import datetime
+import pandas as pd
+
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
This needs to be surrounded by `try-catch` and should appropriate log message denoting the error case.We have to assume that who ever is running did not attempt to setup the `.env` file

Reply:
That is a good one to catch, never thought of that. 
In the next PR, I added a way to check that empty values are not being passed for the API parameters. 

But I will also add in an explicit check for the .env file as well.

Reply:
when a suggestion is made to the PR, usually we tend to resolve in the same PR. if the suggesting is complicated or making it a big Change to the existing PR then we create an issue and say we will resolve in that issue. It's up to you if you want to resolve the issue here or next

Reply:
Got it. Let me go back to this branch and work on this in this PR chain itself then.

Reply:
Commit [7cc642f](https://github.com/tl-its-umich-edu/annoto-gai/pull/8/commits/7cc642fc9d35f9ebc7960c6d21f509086e95a198) addresses this comment,